### ISSUE TITLE
Can this be used alongside MO2?

### ISSUE THREAD

**Issue by spacecowboyx** [pikdum/steam-deck#53]
I have installed it the usual way as per the instructions and it works. I downloaded a massive collection Immersive & Adult - Steam Deck edition and everything was fine. However, I also have MO2 installed via STL. The game through steam is running forced compat with STL to launch the game. If I enable the gui option for MO2 it will eventually open it up where the game can be launched from there using SKSE. 

Otherwise I can skip the launch of MO2 and boot into the game. I have SKSE installed inside the gamedir as well as the mod through this vortex iteration.  I'd like to be able to use Vortex to download the mods and MO2 to manage them but so far I haven't been able to get a seamless experience, especially when it comes to certain scripts and files that technically cannot be "installed" via a mod manager yet Vortex does just fine.  

Am I asking for too much here? If They both cannot be used together on the deck then my next question is do I need to remove the STL forced compat and instead use the SKSE file to launch?

**Comment by pikdum** [comment]
Can MO2 and Vortex even be used alongside each other on Windows?
Since they handle things in very different ways, I'd expect them to be completely incompatible.

**Comment by spacecowboyx** [comment]
From what I've found they can but I'm not sure how to do it. Technically they both can't manage the same game but  merging should be possible. I attempted to import my mods by pointing it at the vortex staging and downloads folder but beyond that I didn't have the metadata and it was too difficult to deal with.  

After a few days of playing with it it works pretty well with some minor hiccups. Don't know if it's related but I think it crashes my input causing none of the deck controls to work sometimes forcing me to bust out my bt keyboard. In addition if I close it the executible is still running in the background so I can't launch it again without killing it.
